cookbooks/logstash/recipes/default.rb

```diff
@@ -1,16 +1,37 @@
 # Logstash recipe
 
 # Create directories
-directory "#{node["logstash"]["dir"]}"
-directory "#{node["logstash"]["dir"]}/conf"
-directory "#{node["logstash"]["dir"]}/data"
+directory "#{node["logstash"]["dir"]}" do
```

Comment:
You could loop through these directories to avoid repetition.

```ruby
# artributes.rb
default[:logstash][:directories] = %w(conf data ...)

# default.rb
node[:logstash][:directories].each do |logstash_dir|
  directory logstash_dir
  # etc
end
```